src/DataService/Player/Player.php

```diff
@@ -887,103 +1001,103 @@ public function setColorSchemeId($colorSchemeId)
      *
      * @return int
      */
-    public function getColumns(): int
+    public function getColumns(): ?int
     {
         return $this->columns;
     }
 
     /**
      * Set the number of columns in the release list.
      *
-     * @param int
+     * @param int $columns
      */
-    public function setColumns($columns)
+    public function setColumns(?int $columns)
     {
         $this->columns = $columns;
     }
 
     /**
-     * Returns Reserved for future use. The default value is false.
+     * Returns reserved for future use. The default value is false.
      *
      * @return bool
      */
-    public function getCompatibilityMode(): bool
+    public function getCompatibilityMode(): ?bool
     {
         return $this->compatibilityMode;
     }
 
     /**
-     * Set Reserved for future use. The default value is false.
+     * Set reserved for future use. The default value is false.
      *
-     * @param bool
+     * @param bool $compatibilityMode
      */
-    public function setCompatibilityMode($compatibilityMode)
+    public function setCompatibilityMode(?bool $compatibilityMode)
     {
         $this->compatibilityMode = $compatibilityMode;
     }
 
     /**
-     * Returns XML layout content for the player control rack.
+     * Returns xML layout content for the player control rack.
      *
      * @return string
      */
-    public function getControlLayoutXml(): string
+    public function getControlLayoutXml(): ?string
     {
         return $this->controlLayoutXml;
     }
 
     /**
-     * Set XML layout content for the player control rack.
+     * Set xML layout content for the player control rack.
```

Comment:
heh, maybe it's better to put all in lowercase

Reply:
So much for my `lcfirst` call. I'll update these and save any fixes in the class generator for the next time I need to regenerate.